.eslintrc.js

```diff
@@ -49,17 +54,18 @@ module.exports = {
 
         // verify every file contains our copyright header
         'header/header': ['error', 'block', [{
-            template: " Copyright (C) 2020 NooBaa ",
+            template: " Copyright (C) 2023 NooBaa ",
```

Comment:
@guymguym do we still need the Copyright template?

Reply:
@dannyzaken I don't know. We can check with an expert if you want to change. But I think you can safely use a regexp to allow any 20** year...

Reply:
https://www.npmjs.com/package/eslint-plugin-header

> When you use a regular expression pattern, you can also provide a template property, to provide the comment value when using eslint --fix:
```
{
    "plugins": [
        "header"
    ],
    "rules": {
        "header/header": [2, "block", [
            {"pattern": " Copyright \\d{4}", "template": " Copyright 2019"}, 
            "My Company"
        ]]
    }
}
```

Reply:
@shirady Can we really just change it like that to 2023? it should make all the existing headers wrong and make eslint error per file... Isn't eslint failing for you now??

Reply:
I change it to be automatic to the current year.

**No header case** - 2 ways to check it:
In VS COde -> create new js file and save it -> Problems "missing header" eslint(header/header) -> Fix this header/header problem -> first line will appear:` /* Copyright (C) 2023 NooBaa */`.
Install eslint globally (brew install eslint)-> `eslint --fix <js file path>` -> first line will appear:` /* Copyright (C) 2023 NooBaa */`.

**Header with the year less than 2023** - I also saved an existing file and run `eslint --fix <js file path>` to see that the year is not changed. for example: `s3rver.js`.

**Illegal header** - added an header` /* Copyright (C) zzz NooBaa */` -> Problems: "incorrenct header eslint(header/header) -> Fix this header/header problem -> first line will appear:` /* Copyright (C) 2023 NooBaa */`.
same result from running `eslint --fix <js file path>`.

Reply:
oh sorry I didn't see the full diff and missed that we already have a pattern.
in any case, looks good now.